src/h3/geo.pyx

```diff
@@ -112,6 +177,56 @@ def polyfill(geos, int res):
     return mv
 
 
+def polyfill_geojson(geojson, int res=9):
+    """ Set of hexagons whose center is contained in a GeoJson Polygon object.
+
+    The polygon is defined exactly as in the GeoJson standard, so
+    `geojson` should be a dictionary like:
+    {
+        'type': 'Polygon',
+        'coordinates': [...]
+    }
+
+    'coordinates' should be a list of LinearRings, where the first ring describes
+    the exterior boundary of the Polygon, and any subsequent LinearRings
+    describe holes in the polygon.
+
+    Note that we don't provide an option for the order of the coordinates,
+    as the GeoJson standard requires them to be in lng/lat order.
+
+    Parameters
+    ----------
+    geojson : dict
+    res : int
+        The resolution of the output hexagons
+    """
+
+    # todo: this one could handle multipolygons...
```

Comment:
Or feature collections, etc. In JavaScript, we moved this functionality to an entirely separate library (https://github.com/uber/geojson2h3).

Reply:
The more I think about it, the more I'd leave this functionality to a separate lib. There's a fair range of stuff you might want to address w/r/t geojson, including `h3_to_multipolygon` output.

Reply:
Hmm. So the current `h3-py` `master` branch has the `h3_set_to_multi_polygon()` function. If we want feature parity, the `cython` branch will have to provide that as well. (I was planning on adding that next.)

Do we need this branch to be fully backwards compatible in terms of `h3_set_to_multi_polygon()`?

Reply:
`h3_set_to_multi_polygon` is just a reasonable binding representation of `h3SetToLinkedGeo`, which is part of the core C API. The pieces that go beyond the C API are:

**GeoJSON to H3**
- Accepting `MultiPolygon` input coordinates
- Accepting GeoJSON input rather than just coords
- Accepting `FeatureCollection` input

**H3 to GeoJSON**
- Outputting GeoJSON rather than just coords
- Outputting alternative transformations, e.g. a `FeatureCollection` of hexagons

Basically anything that knows about *actual* GeoJSON formats, rather than just coordinates, seems out of scope IMO - that's why we put this in a separate lib for JS.

Reply:
I do like the idea of keeping this library minimal. I'll leave these in for now so that we match the current `master` branch functionality, but maybe we can revisit which functions we want to include in 4.0.